projects/demo/src/modules/components/breadcrumbs/examples/2/index.html

```diff
@@ -0,0 +1,45 @@
+<tui-breadcrumbs [style.color]="'red'">
+    <tui-hosted-dropdown
+        *tuiItem
+        [content]="content"
+    >
+        <button
+            tuiIconButton
+            icon="tuiIconMoreHorizontal"
+            size="xs"
+            appearance="secondary"
+            title="More"
+        ></button>
+        <ng-template #content>
+            <tui-data-list>
+                <a
+                    *tuiRepeatTimes="let index of items.length - max"
+                    tuiOption
+                    [href]="items[index].link"
+                >
+                    {{ items[index].caption }}
+                </a>
+            </tui-data-list>
+        </ng-template>
+    </tui-hosted-dropdown>
+    <ng-container *tuiRepeatTimes="let index of max">
+        <a
+            *tuiItem
+            tuiLink
+            [routerLink]="items[index + items.length - max].link"
```

Comment:
If button is not inside "More" button, it leads to nowhere.

Probably we should use `href` too.

https://github.com/Tinkoff/taiga-ui/assets/35179038/222a7af0-fc4f-4dd3-b262-9f57c1e26feb

Reply:
Oops, my bad